tests/features/drupal_mail.feature

```diff
@@ -0,0 +1,20 @@
+@api @internalMail
+Feature: Drupal Mail features
+  In order to check if a mail was sent correctly
+  As an Administrator
+  I want to be able to catch the mail and verify its properties
+
+  Background:
+    Given platform is configured to use the internal mail handling
+
+  Scenario: Checking mail action for requesting new password.
```

Comment:
I would change  "_Checking mail action for requesting new password._" for "_Checking mail action for requesting new password **by using the username as identifyer**._"